packages/elements/src/overlay/managers/interaction-lock-manager.ts

```diff
@@ -162,12 +162,12 @@ export class ScrollLockManager {
    */
   private saveScrollPosition(): void {
     if (document.scrollingElement) {
-      this.scrollTop = document.scrollingElement.scrollTop;
-      this.scrollLeft = document.scrollingElement.scrollLeft;
+      this.scrollTop = document.scrollingElement?.scrollTop || 0;
+      this.scrollLeft = document.scrollingElement?.scrollLeft || 0;
     } else {
       // Since we don't know if is the body or html, get max.
-      this.scrollTop = Math.max(document.documentElement.scrollTop, document.body.scrollTop);
-      this.scrollLeft = Math.max(document.documentElement.scrollLeft, document.body.scrollLeft);
+      this.scrollTop = Math.max(document.documentElement?.scrollTop, document.body?.scrollTop) || 0;
+      this.scrollLeft = Math.max(document.documentElement?.scrollLeft, document.body?.scrollLeft) || 0;
```

Comment:
`Math.max(10, undefined)` returns `NaN` instead of `10`.
```suggestion
      this.scrollTop = Math.max(document.documentElement.scrollTop, document.body?.scrollTop ?? 0);
      this.scrollLeft = Math.max(document.documentElement.scrollLeft, document.body?.scrollLeft ?? 0);
```